scripts/lbnl_group.nhc

```diff
@@ -0,0 +1,40 @@
+# LBNL NHC - Group/members check
+#
+# Andy Georges <andy.georges@ugent.be>
+# 20 April 2018
+
+GETENT_CMD="${GETENT:-getent}"
+
+# syntax: check_group_member [-t timeout] -g group
+function check_group_member() {
+    local TIMEOUT=10
+    local GROUP="" WHOLEGROUP=""
+
+    OPTIND=1
+    while getopts ":t:g:" OPTION; do
+        case "$OPTION" in
+            t) TIMEOUT="$OPTARG" ;;
+            g) GROUP="$OPTARG" ;;
+            -) ((OPTIND++)) ; break ;;
+            :) die 1 "$CHECK: Option -$OPTARG requires an argument."; return 1 ;;
+            \?) die 1 "$CHECK: Invalid option: -$OPTARG"; return 1 ;;
+        esac
+    done
+
+    WHOLEGROUP=$($GETENT_CMD group "$GROUP" 2>/dev/null)
+    if [ $? -ne 0 ]; then
+            die 1 "$CHECK: group $GROUP not found"
+            return 1
+    fi
+
+    for USER in $(echo "$WHOLEGROUP" | cut -d":" -f 4 | tr "," " "); do
```

Comment:
`USER` is a builtin bash variable. choose another one

Reply:
Done.